### ISSUE TITLE
devtools install

### ISSUE THREAD

**Issue by trafficonese** [region-spotteR/PrepPlot#10]
Since PrepPlot is not a package, it is not possible to install it via devtools. 

The code to install via devtools would be:
devtools::install_github("region-spotteR/PrepPlot")

but this error appears:
Installation failed: Does not appear to be an R package (no DESCRIPTION)

So I think the only way to get the code is either download it or copy/paste the function

**Comment by region-spotteR** [comment]
Yeah at the time I made the package, I had no time to fix that issue. But I will update the package now anyways, then there will be `devtools`-support as well
